recipes/sphinxcontrib-paverutils/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sphinxcontrib-paverutils" %}
+{% set version = "1.14.0" %}
+{% set sha256 = "2124c851f96356012ed5c92ce320ca3a842cce8a94ddd2319bcb0835b48b9f80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please change to

`url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`